src/renderer/screens/exchange/Swap2/utils/shared/getAccountLikeId.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import type { AccountLike } from "@ledgerhq/live-common/lib/types";
+
+const getAccountId = (fromAccount: ?AccountLike) => {
+  if (!fromAccount) return "";
```

Comment:
I doubt you need this helper, you have the `getAccountCurrency` helper which will give you the token or currency already, and you can get the id from there directly. Don't worry about `ChildAccount` because they will not be swappable

Reply:
Oh right, thanks, I'll give a try to `getAccountCurrency`

Reply:
Change here [fa0a5290a1de93c9a89a4577b6b6884412361341](https://github.com/juan-cortes/ledger-live-desktop/pull/14/commits/fa0a5290a1de93c9a89a4577b6b6884412361341)